packages/services/ds/src/ops/server/list_for_env.rs

```diff
@@ -8,6 +8,7 @@ pub struct Input {
 	pub tags: HashMap<String, String>,
 	pub include_destroyed: bool,
 	pub cursor: Option<Uuid>,
+	pub limit: usize,
```

Comment:
these limits need some sort of cap on it

Reply:
This is an internal op, the limits should go on the api layer